CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Contributing to Spyder-IDE
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+## General Guidelines
+
+This page documents at a very high level how to contribute to Spyder.
+Please Check the [Spyder-IDE Contributor Documentation](https://github.com/spyder-ide/spyder/wiki/Contributing-to-Spyder) for a more to developing and contributing to the Spyder.
+
+
+## Setting Up a Development Environment
+
+
+### Cloning the repo
+
+```bash
+  $ git clone https://github.com/spyder-ide/spyder.git
+```
+
+### Creating a conda environment (or virtualenv)
+
+```bash
+  $ conda create -n spyder spyder
```

Comment:
should I leave this instruction? or only advice to install dependencies from requirements file?